src/apps/explorer/components/TransanctionBatchGraph/index.tsx

```diff
@@ -208,23 +210,32 @@ function TransanctionBatchGraph({
   const [elements, setElements] = useState<ElementDefinition[]>([])
   const cytoscapeRef = useRef<Cytoscape.Core | null>(null)
   const cyPopperRef = useRef<PopperInstance | null>(null)
+  const [resetZoom, setResetZoom] = useState<boolean | null>(null)
   const theme = useTheme()
-  const heightSize = window.innerHeight - HEIGHT_HEADER_FOOTER
+  const { innerHeight } = useWindowSizes()
+  const heightSize = innerHeight && innerHeight - HEIGHT_HEADER_FOOTER
   const setCytoscape = useCallback(
     (ref: Cytoscape.Core) => {
       cytoscapeRef.current = ref
-      cytoscapeRef.current.layout(getLayout()).run()
-      cytoscapeRef.current.fit()
+      const updateLayout = (): void => {
+        ref.layout(getLayout()).run()
+        ref.fit()
+      }
+      ref.on('resize', () => {
```

Comment:
@henrypalacios is this subscription properly cleaned when the chart is unmounted?

Reply:
The listeners seem to be removed when unmounted the component, (I couldn't catch it from the documentation). 
But with a thorough review of the `Cy.Core` object I got:

![Selection_558](https://user-images.githubusercontent.com/4270166/171553397-f633c868-bede-4cc9-93be-2d850f6a3d0e.png)


1. I found that there were memory leaks for the events at interaction time.

2. An untyped method that [**remove** all the listener](https://github.com/cowprotocol/explorer/blob/497cc3daf6e795bdd9bb7ffc1812b04bc8b5b2b9/src/apps/explorer/components/TransanctionBatchGraph/index.tsx#L259) that we can use it to make sure that it is going to clean everything when disassembling.